### ISSUE TITLE
no go files to analyze

### ISSUE THREAD

**Issue by samsondav** [reviewdog/action-golangci-lint#28]
Running on [the chainlink repo](https://github.com/smartcontractkit/chainlink) we get:

```
level=error msg="Running error: context loading failed: no go files to analyze"
level=error msg="Timeout exceeded: try increasing it by passing --timeout option"
```

Is this a bug or a misconfiguration?

**Comment by Wollac** [comment]
I had a similar issue and for me increasing the timeout in `golangci_lint_flags` worked. Seams like golangci is reporting a strange error about "no go files to analyze" on timeout...

**Comment by haya14busa** [comment]
https://github.com/reviewdog/action-golangci-lint/pull/46 may fix the issue.

Also, if it's golangci-lint issue, it's not issue of this actions. Can you try the new version and/or try golangci-lint itself as well if it's still reproducible?
